code/game/objects/items/weapons/RCD.dm

```diff
@@ -333,7 +333,7 @@ GLOBAL_LIST_INIT(rcd_door_types, list(
 					return FALSE
 				playsound(loc, usesound, 50, 1)
 				var/turf/AT = A
-				AT.ChangeTurf(/turf/space)
+				AT.ChangeTurf(AT.baseturf)

```

Comment:
No longer shall RCD's possess the supernatural ability to bore through an entire planet's crust, straight through the core, to outer space.